v1_20_R4/src/main/java/net/citizensnpcs/nms/v1_20_R4/util/NMSImpl.java

```diff
@@ -1577,6 +1618,11 @@ public void setKnockbackResistance(org.bukkit.entity.LivingEntity entity, double
 
     @Override
     public void setLocationDirectly(org.bukkit.entity.Entity entity, Location location) {
+        // Todo temp teleport
```

Comment:
Is there any equivalent method in folia?

Reply:
Not on Folia, because it is used directly in Folia's NMS.